setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
It's still releasing 0.y.z, which, according to semvar means

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

We may be using it in production, but I think there are plenty of APIs in fedmsg that should change before we claim it's stable.